src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -86,8 +86,14 @@
 
   private final ImportSectionTransformer importSectionTransformer;
 
-  // Whether the initialization code should include user-facing comments like TODOs. This should be
-  // false if the initialization code is being used to generate tests, rather than code samples.
+  // Whether the initialization code should include hardcoded comments like TODOs. This
+  // should only be true when generating incode samples.
+  //
+  // This field should be set to false when generating tests because tests do not need to include
+  // user facing comments.
+  //
+  // This field should be set to false when generating standalone samples because comments in
+  // standalone samples are derived from user configurations, not hardcoded.
   private final boolean generateUserFacingComments;
```

Comment:
minor: Since we're talking about a distinction between these non-configurable comments and the ones that the user specifies, consider renaming this field and associated params. WDYT? Maybe `generateStandardComments`?

(I wonder what we meant originally by "user facing" comments. In the gapic-gen context, all comments we generate are user-facing....)

Reply:
Sounds good.